### ISSUE TITLE
I had a problem with customizing image flicker on the cell when refreshing uitableview

### ISSUE THREAD

**Issue by 835932621** [SDWebImage/SDWebImage#2735]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | iOS 
 Platform Version        | 10.3.3
 SDWebImage Version      |  5.0.0
 Integration Method      |  cocoapods 
 Xcode Version           |  Xcode 10.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | yes
 Demo project link       | e.g. link to a demo project that highlights the issue

I had a problem with customizing image flicker on the cell when refreshing uitableview, for example
[self.pic sd_setImageWithURL:[NSURL URL WithString: picUrlStr] placeholder Image:[UIImage imageNamed:@ "default_load_image"];

**Comment by kinarob** [comment]
@835932621 You can see [#2175](https://github.com/SDWebImage/SDWebImage/pull/2175) for more info. dreampiggy explained in great detail.

**Comment by 835932621** [comment]
Thank you very much. The problem I encountered was partial scintillation. Now it's solved, because the background gives us a picture with very large pixels, but the pixels don't change after shrinking, which leads to local flicker.
